rclcpp/include/rclcpp/node.hpp

```diff
@@ -74,46 +82,46 @@ class Node
     std::string topic_name,
     size_t queue_size,
     std::function<void(const std::shared_ptr<MessageT> &)> callback,
-    rclcpp::callback_group::CallbackGroup::SharedPtr group=nullptr);
+    rclcpp::callback_group::CallbackGroup::SharedPtr group = nullptr);
 
   /* Create a timer. */
   rclcpp::timer::WallTimer::SharedPtr
   create_wall_timer(
     std::chrono::nanoseconds period,
     rclcpp::timer::CallbackType callback,
-    rclcpp::callback_group::CallbackGroup::SharedPtr group=nullptr);
+    rclcpp::callback_group::CallbackGroup::SharedPtr group = nullptr);
 
   rclcpp::timer::WallTimer::SharedPtr
   create_wall_timer(
     std::chrono::duration<long double, std::nano> period,
     rclcpp::timer::CallbackType callback,
-    rclcpp::callback_group::CallbackGroup::SharedPtr group=nullptr);
+    rclcpp::callback_group::CallbackGroup::SharedPtr group = nullptr);
 
   typedef rclcpp::callback_group::CallbackGroup CallbackGroup;
   typedef std::weak_ptr<CallbackGroup> CallbackGroupWeakPtr;
   typedef std::list<CallbackGroupWeakPtr> CallbackGroupWeakPtrList;
 
   /* Create and return a Client. */
-  template <typename ServiceT>
+  template<typename ServiceT>
   typename rclcpp::client::Client<ServiceT>::SharedPtr
   create_client(
     std::string service_name,
-    rclcpp::callback_group::CallbackGroup::SharedPtr group=nullptr);
+    rclcpp::callback_group::CallbackGroup::SharedPtr group = nullptr);
 
   /* Create and return a Service. */
-  template <typename ServiceT>
+  template<typename ServiceT>
   typename rclcpp::service::Service<ServiceT>::SharedPtr
   create_service(
     std::string service_name,
     std::function<void(const std::shared_ptr<typename ServiceT::Request> &,
-                       std::shared_ptr<typename ServiceT::Response>&)> callback,
-    rclcpp::callback_group::CallbackGroup::SharedPtr group=nullptr);
+    std::shared_ptr<typename ServiceT::Response> &)> callback,
+    rclcpp::callback_group::CallbackGroup::SharedPtr group = nullptr);
```

Comment:
This one also doesn't look right, I would have expected:

```c++
create_service(
    std::string service_name,
    std::function<void (const std::shared_ptr<typename ServiceT::Request> &,
        std::shared_ptr<typename ServiceT::Response>&)> callback,
    rclcpp::callback_group::CallbackGroup::SharedPtr group=nullptr);

```

Or...

```c++
create_service(
    std::string service_name,
    std::function<void (
        const std::shared_ptr<typename ServiceT::Request> &,
        std::shared_ptr<typename ServiceT::Response>&)> callback,
    rclcpp::callback_group::CallbackGroup::SharedPtr group=nullptr);

```

Regardless of what I expected, the result from the linter is pretty confusing to me, since parameters to nested items end up on the same line.